pkg/storage/redis/redis.go

```diff
@@ -182,13 +257,68 @@ func (backend *Backend) Put(ctx context.Context, record *databroker.Record) (err
 				Score:  float64(version),
 				Member: bs,
 			})
+			p.ZAdd(ctx, getRecordTypeChangesKey(record.GetType()), &redis.Z{
+				Score:  float64(record.GetModifiedAt().GetSeconds()),
```

Comment:
is there a reason to limit resolution to just seconds?

Reply:
Because it was a simple method call. I guess we can use something else, though I don't know what the advantage would be.

Reply:
there might be multiple records arriving at the very same second